src/main/jupyter/oss/security/SecurityRatingAnalysis.ipynb

```diff
@@ -4390,7 +4390,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.1"
+   "version": "3.6.9"
```

Comment:
Was there any issue in python version 3.7.1?

Reply:
I didn't notice any issue. Actually I didn't update this field. Looks like Jupyter updated it. To be honest, I am really familiar with the structure of `ipynb` files. Do you think this update can cause some problems?

Reply:
I'll just revert this update. I think I updated the notebook in #652 and committed this update here by mistake.